mandatory/4-tax.js

```diff
@@ -5,7 +5,10 @@
   Sales tax is 20% of the price of the product.
 */
 
-function calculateSalesTax() {}
+function calculateSalesTax(productPrice) {
+ let productTotal=productPrice+(productPrice*0.2);
```

Comment:
You have no spaces between the variable names and the expression on the right hand side. Double check the `README` for instructions on how to format your code